cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -1,151 +1,54 @@
-
-    # remove 'ethnicity' label (needs to be done separately since its no longer in the schema definition)
-    if "ethnicity" in dataset.obs:
-        del dataset.obs["ethnicity"]
-
-    # Update deprecated ontologies with known direct replacements
-    disease_ontology_update_map = {
-        "MONDO:0008345": "MONDO:0800029",
-        "MONDO:0004553": "MONDO:0017853",
-    }
-    cell_type_ontology_update_map = {
-        "CL:0002609": "CL:0010012",
-        "CL:0011107": "CL:0000636",
-    }
-    assay_ontology_update_map = {
-        "EFO:0030002 (BD Rhapsody)": "EFO:0700003",
-        "EFO:0010183 (BD Rhapsody)": "EFO:0700003",
-    }
-
-    def update_categorical_column_vals(dataframe, column_name, update_map):
-        if dataframe[column_name].dtype != "category":
-            dataframe[column_name] = dataframe[column_name].astype("category")
-        for deprecated_term, new_term in update_map.items():
-            if deprecated_term in dataframe[column_name].cat.categories:
-                # add new one if not already in category, else continue
-                if new_term not in dataframe[column_name].cat.categories:
-                    dataframe[column_name] = dataframe[column_name].cat.add_categories(new_term)
-                # replace in dataset
-                dataframe.loc[dataframe[column_name] == deprecated_term, column_name] = new_term
-                # remove deprecated_term from category
-                dataframe[column_name] = dataframe[column_name].cat.remove_categories(deprecated_term)
-
-    update_categorical_column_vals(dataset.obs, "disease_ontology_term_id", disease_ontology_update_map)
-    update_categorical_column_vals(dataset.obs, "cell_type_ontology_term_id", cell_type_ontology_update_map)
-    update_categorical_column_vals(dataset.obs, "assay_ontology_term_id", assay_ontology_update_map)
-
-    # Set suspension type
-
-    # mappings of assays (or assays + child term assays) to corresponding suspension_type
-    # valid assays with multiple possible suspension_types shown but commented out
-    match_assays = {
-        # 'EFO:0010010': ['cell', 'nucleus'],
-        "EFO:0008720": "nucleus",
-        # 'EFO:0008722': ['cell', 'nucleus'], ,
-        "EFO:0030002": "cell",
-        "EFO:0008853": "cell",
-        "EFO:0030026": "nucleus",
-        # 'EFO:0010550': ['cell', 'nucleus'],
-        "EFO:0008919": "cell",
-        "EFO:0008939": "nucleus",
-        "EFO:0030027": "nucleus",
+    # ONTOLOGY TERMS TO UPDATE ACROSS ALL DATASETS IN CORPUS
+    # Initialization is AUTOMATED for newly deprecated terms that have 'Replaced By' terms in their ontology files
+
+    # Curators should review the monthly 'Curator Report' and add deprecated term replacements to corresponding map if
+    # 'Replaced By' is not available for a deprecated term.
+
+    # If Curators have non-deprecated term changes to apply to all datasets in the corpus where applicable,
+    # add them here.
+    ontology_term_maps = {
```

Comment:
If we move this out of the function we could mock it for better testings.

Reply:
I like having the unit test set up to run the migrate script as written without mocking. Especially since Lattice may need to modify this map after it's generated. At a minimum, it ensures a test run that would catch any syntax errors leading to an exception before merging. 

That said, still a good suggestion so we can mock it for other additional tests. I'll go ahead and merge for now but keeping that in mind for another pass